scripts/retrieve_seawater_temperature.py

```diff
@@ -87,13 +87,13 @@
         # Spatial coverage: European waters (-12°W to 42°E, 33°N to 72°N)
         # Depth range: 5-15m (suitable for heat pump intake depths)
         logger.info(
-            f"Downloading seawater temperature data for year {snakemake.wildcards.year}"
+            f"Downloading seawater temperature data for year {snakemake.wildcards.planning_horizons}"
         )
 
         _ = copernicusmarine.subset(
             dataset_id="cmems_mod_glo_phy_my_0.083deg_P1D-m",  # Global ocean physics reanalysis
-            start_datetime=f"{snakemake.wildcards.year}-01-01",
-            end_datetime=f"{int(snakemake.wildcards.year)}-12-31",
+            start_datetime=f"{snakemake.wildcards.planning_horizons}-01-01",
+            end_datetime=f"{int(snakemake.wildcards.planning_horizons)}-12-31",
```

Comment:
Should be `{year}` as it doesn't refer to the `{planning_horizon}`.
```suggestion
            start_datetime=f"{snakemake.wildcards.year}-01-01",
            end_datetime=f"{int(snakemake.wildcards.year)}-12-31",
```